docs/event.md

```diff
@@ -94,7 +88,7 @@ Setting rotation and zoom to a moon object with drag and drop and mousewheel eve
 
     //create and assign variables to objects...
 
-    LumaGL.Events.create(app, {
+    LumaGL.Events.create(gl, {
```

Comment:
To make justice to our ES6 port, I would prefer that we use the namespace less version in examples:
```
import {Events} from 'luma.gl';
Events.create(gl, ...
```